ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.1.yaml

```diff
@@ -0,0 +1,192 @@
+base_images:
```

Comment:
We already have 4.1->4.15 upgrade which contains the commit history of what/when changes are introduced.
If we create a new 4.1->4.16 in this PR, and then use another PR to remove the old 4.1->4.15 upgrade, then we are very likely to lost the commit history since git/github do not have the knowledge to track those two files as the purpose of 4.1-latest version chain upgrade.
So, IMO, we should bump up the 4.1->4.15 upgrade to 4.1->4.16 upgrade.
Or if you think bump up is hard to maintain and you prefer copy/paste and then remove old one, it's better to do them in one PR to let git/github know the relationship between those two files.

Reply:
Good point, I also do them in one PR, just need to move the new one to overwrite the old one.